test/step.test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const stepThroughExpression = require('../lib/simplifyExpression/stepThrough');
+
+function testStepsInExpression(exprStr, outputStr) {
+  it ('test the steps in simplifying expression', function () {
+    assert.equal((stepThroughExpression(math.parse(exprStr))[0]['changeType']), outputStr);
```

Comment:
Just a question ! For eg - if the input expression is `2(x + y)`and running `stepThroughExpression` gives the new_node. But what should be tested next!? changeGroup? Value? or the args?

This is the output 
```
Node {
  implicit: true,
  op: '*',
  fn: 'multiply',
  args: 
   [ Node { value: '2', valueType: 'number', changeGroup: undefined },
     Node { name: 'x', changeGroup: undefined } ],
  changeGroup: 1 }
```